Atomic/util.py

```diff
@@ -635,3 +647,64 @@ def get_signature_read_path(reg_info):
     # Return the defined path for where signatures should be read
     # or none if no entry is found
     return reg_info.get('sigstore', None)
+
+def remove_port(_input):
```

Comment:
Maybe this should be called strip_port_text or something like that, I see remove_port, and I think you are changing something in the configuration.